### ISSUE TITLE
Is 'tf_val' used incorrectly for 'train_dataset'

### ISSUE THREAD

**Issue by keysking** [jeya-maria-jose/Medical-Transformer#22]
On line 87 of train.py and 82 of test.py
```python
tf_train = JointTransform2D(crop=crop, p_flip=0.5, color_jitter_params=None, long_mask=True)
tf_val = JointTransform2D(crop=crop, p_flip=0, color_jitter_params=None, long_mask=True)
train_dataset = ImageToImage2D(args.train_dataset, tf_val)  # Here
val_dataset = ImageToImage2D(args.val_dataset, tf_val)
```

**Comment by charleneolive** [comment]
I agree with you.  @keysking I wonder why it should not be tf_train? @jeya-maria-jose . Did you mean to add horizontal flips as augmentations.

**Comment by jeya-maria-jose** [comment]
Fixed #31
